Rewrite known-issues.rst

Mrt134 opened the pull request:
- PEP8
- Add extensive Sphinx/ReST markup.
- Rewrite sentences for clarity, grammar, spelling, etc.
- Build test - no errors.

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

Mrt134 committed:
Rewrite known-issues.rst

- PEP8
- Add extensive Sphinx/ReST markup.
- Rewrite sentences for clarity, grammar, spelling, etc.
- Build test - no errors.

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
 ZFS snapshot creation and rollback is not affected.
 
-To workaround this issue iocage 1.6.0 introduced a ``hack88`` property.
+To work around this issue, **iocage 1.6.0** introduced a **hack88**
```
`hack88` no longer exists. We suggest the user either supplies their own `--name`, or use `--short` if that functionality is needed.

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
 Shut down jail:
 
-``iocage stop myjail``
+:samp:`# iocage stop myjail`
```
Remove all `hack88` stuff above and below.

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
+**hack88=0**.
 
-To create a system wide default (introduced in 1.6.0) for all newly created jails use:
+Create a system wide default (introduced in **1.6.0**) for all newly
```
This is also no longer needed, you cannot set short for all jails by default, it needs to be specified. Also removing old version numbers.

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
 
-To create a system wide default (introduced in 1.6.0) for all newly created jails use:
+Create a system wide default (introduced in **1.6.0**) for all newly
+created jails with :command:`iocage set hack88=1 default`
```
Same as above, nukkke!

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
 
-iocage does not validate properties right now. Please refer to man page to see what is supported
-for each property. By default iocage pre-configures each property with a safe default.
+:command:`iocage` does not currently validate properties. Please refer
```
We can remove this section, as I do validate properties now.

skarekrow commented on doc/source/known-issues.rst:
```diff
@@ -1,97 +1,126 @@
-As a workaround iocage allows a warm restart without destroying the jail.
-By default the restart sub-command will execute a warm restart.
+As a workaround, :command:`iocage` allows a soft restart without
+destroying the jail. By default, :command:`iocage restart` executes a
```
No longer done by default, user must supply `-s`, `iocage restart -s UUID | NAME`

Mrt134 committed:
Update with review comments

Remove hack88 material.
Remove validate properties section.
Rework soft jail restart material.
